src/Chainweb/SPV/CreateProof.hs

```diff
@@ -480,7 +481,7 @@ crumbsToChain db srcCid trgHeader
         let !adjIdx = fromJuste $ blockHashRecordChainIdx (view blockAdjacentHashes cur) h
         go adjpHdr t ((adjIdx, cur) : acc)
 
-minimumTrgHeader
+findTrgHeader
```

Comment:
I think, the old name is still correct and more meaningful. Maybe call it `minimumValidTrgHeader`?